wfe/wfe.go

```diff
@@ -1048,9 +1048,19 @@ func (wfe *WebFrontEndImpl) verifyOrder(order *core.Order) *acme.ProblemDetails
 	}
 	// Check that all of the identifiers in the new-order are DNS type
 	for _, ident := range idents {
+		//ip check, we'll need to check for exclude private IPs on production, but I think this works for pebble
```

Comment:
There's no need for this check in Pebble.

Reply:
yes so I didn't included it there

Reply:
I would remove this comment, or heavily rewrite it, since in the current way it's pretty confusing (especially who "I" is).

Reply:
(In this PR, it's clear, but once this is merged, it's confusing :) )